oshdb-api/src/test/java/org/heigit/bigspatialdata/oshdb/api/tests/TestOSMDataFilters.java

```diff
@@ -139,6 +142,18 @@ public void tagList() throws Exception {
         .timestamps(timestamps1)
         .count();
     assertEquals(5, result.intValue());
+    // only tags
```

Comment:
this should probably read "tags and keys mixed"

Reply:
great. This is what I asked myself. So I can use them mixed and they will be OR joined. so limiting to "landuse"="*" OR "natural"="forest" is now possible.

Reply:
exactly